astropy/coordinates/jparser.py

```diff
@@ -0,0 +1,81 @@
+import re
+
+import numpy as np
+
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+
+
+def _sexagesimal(g):
+    # convert matched regex groups to sexigesimal array
+    sign, h, m, s, frac = g
+    sign = [1, -1][sign == '-']
```

Comment:
Don't use implicit bool => int casting.  Instead use Python ternary `sign = -1 if (sign == '-') else 1`.  (The parens in that expression are not necessary but I find that it does improve readability).

Reply:
There is no type casting happening here afaict since `int` inherits from `bool`, but I do get the point on readability.

Reply:
Interesting, I didn't know that (though it is the other way around, `bool` inherits from `int`).  I guess that's the Python hack for making it easy to interchange between True/False and 1/0 in code.